README.md

```diff
@@ -1101,19 +1107,27 @@ EXAMPLES
 ```
 generate xccdf_benchmark2inspec_stub              Translate a DISA STIG XCCDF Benchmark XML file into a skeleton for an InSpec profile
 
-  USAGE
-    $ saf generate xccdf_benchmark2inspec_stub -i <stig-xccdf-xml> -o <output-folder> [-h] [-m <metadata-json>] [-s] [-r | -S] [-l <line-length>] [-e]
+USAGE
+  $ saf generate xccdf_benchmark2inspec_stub -i <stig-xccdf-xml> -o <output-folder> [-h] [-m <metadata-json>] [-T vuln|group|cis|stig] [-s] [-L info|warn|debug|verbose]
```

Comment:
why are we changing the internal naming, but not the external function name?

Reply:
Done